joomla/joomla-cms
# [5] [draft] Displayed of field values in the table of the list of articles in the admin panel.

### Summary of Changes

New feature. Allows you to display the values of custom fields in the cells of the table of the list of articles. Enabled for each field separately.

This PR is made only for articles and without the use of localization. As soon as we come to the correct code, I will supplement the localization and do the same for Users.

![изображение](https://github.com/joomla/joomla-cms/assets/6898474/cfe8ae05-a641-45c0-8213-ddb65ee2da91)

### Testing Instructions
1. Adding a column to a database table
```SQL
ALTER TABLE `#__fields` ADD `show_in_list` BOOLEAN NOT NULL DEFAULT FALSE AFTER `only_use_in_subform` ; 
```
AND add code in **`\media\system\js\table-columns.js`** in 11 line.
```JS
this.listOfHidden = this.$headers.map((th, i) => th.classList.contains('d-hide') ? i : false).filter((i) => i !== false);
```
2. Create a custom field in the admin panel, and turn on the "In list administration page" mode
3. Create and Safe articles and fill in custom fields for articles.
4. You open a page with a table of the list of materials, and then you see columns with the values of user fields.

### Actual result BEFORE applying this Pull Request

Previously, the values of user fields were not displayed.

### Expected result AFTER applying this Pull Request

And now you can see the values of custom fields.

This is necessary to view prices, volume, quantity, and other data in the form of a table. It is very important to be able to see all the values at once.


### Link to documentations
Please select:



_Work has also been done to filter the display of fields configured for certain categories. That is, the fields will not be displayed in the table if the view does not touch the desired categories._

![изображение](https://github.com/joomla/joomla-cms/assets/6898474/88a8b516-c176-49ac-84dd-7f79bab9d484)
